ArticleTemplates/collectionTemplate.html

```diff
@@ -28,6 +28,7 @@
         GU.bootstrap.init({             
             asyncStylesFilename: "__ASYNC_STYLES__",
             fontSize: "__FONT_SIZE__",
+            lineHeight: "__LINE_HEIGHT__",
```

Comment:
You've removed `lineHeight: "__LINE_HEIGHT__"` from `articleTemplate.html`, I'd also do the same to the remaining instances in the HTML templates.